op-deployer/pkg/deployer/flags.go

```diff
@@ -97,6 +101,28 @@ var (
 		EnvVars: PrefixEnvVar("INTENT_CONFIG_TYPE"),
 		Value:   string(state.IntentConfigTypeStandard),
 	}
+	EtherscanAPIKeyFlag = &cli.StringFlag{
+		Name:     EtherscanAPIKeyFlagName,
+		Usage:    "etherscan API key for contract verification.",
+		EnvVars:  PrefixEnvVar("ETHERSCAN_API_KEY"),
+		Required: true,
+	}
+	ContractBundleFlag = &cli.StringFlag{
+		Name:    ContractBundleFlagName,
+		Usage:   "contract bundle/grouping to verify (superchain, opchain, or implementations)",
+		EnvVars: PrefixEnvVar("CONTRACT_BUNDLE"),
+	}
+	ContractNameFlag = &cli.StringFlag{
+		Name:    ContractNameFlagName,
+		Usage:   "contract name (mathcing a field within state.json) to verify",
+		EnvVars: PrefixEnvVar("CONTRACT_NAME"),
+	}
+	L2ChainIndexFlag = &cli.IntFlag{
```

Comment:
Let's use the chain ID instead of index since most folks won't remember the index. There are helper methods to select the right chain.

Reply:
Done in https://github.com/ethereum-optimism/optimism/pull/14359/commits/6f7f3d00f287422c58d4dd2b4530996e4a735a09